### PR TITLE
fix(card): fix no images appearing on some cards on home screen

### LINKED ISSUE #708: Next up covers aren't loaded
I don't get any network or console error, so I think the card just doesn't displays the image.
print_screen_2021-02-11-16-15-32

### PR DESCRIPTION
**Pull request by Camc314**
Fixes #708

### CODE REVIEW

File: components/Item/Card/Card.vue [hunk 1]
```diff
@@ -237,13 +237,8 @@ export default Vue.extend({
...
     },
     getImageType(): ImageType {
-      if (
-        this.shape === 'thumb-card' &&
-        !['CollectionFolder', 'Folder', 'UserView'].includes(
-          this.item?.Type || ''
-        )
-      ) {
-        return ImageType.Thumb;
+      if (this.shape === 'thumb-card' && this.item.Type === 'Movie') {
```
Comment (ThibaultNocchi) [inline components/Item/Card/Card.vue:240]: How does that work for series, do they use the Primary?

Comment (Camc314) [reply components/Item/Card/Card.vue:240]: Do you mean for episodes?

If I remember correctly,
Episodes have primary image type that are thumb shaped

Comment (ThibaultNocchi) [reply components/Item/Card/Card.vue:240]: Yeah episodes, my bad. And that's why it worked with my episodes in next up :p

Comment (ThibaultNocchi) [reply components/Item/Card/Card.vue:240]: A thumb being these wide images? And posters the vertical ones IIRC?

**Review by ThibaultNocchi — APPROVED** [review]

File: components/Item/Card/Card.vue (hunk 1 above)
Comment (Camc314) [reply components/Item/Card/Card.vue:240]: Yeah that's right

**Comment by ferferga** [comment]
How are CollectionFolder, Folder and UserView types handled now?

**Comment by Camc314** [comment]
I'm going to double check. But I think It's unchanged

**Review by ferferga — APPROVED** [review]

**Pull request merged**